### PR TITLE
Update test format, run ember-cli-update

### PR DESCRIPTION
**Pull request by rwwagner90**
This updates the test format to the new testing style, runs ember-cli-update, removes jQuery, enables Octane, etc.

I also removed `node-tests` as it did not seem to be used. Please correct me if I am wrong there.

### CODE REVIEW

**Comment by rwwagner90** [comment]
Travis still doesn't seem to be reporting here, but the build appears to be running here https://travis-ci.org/github/ember-cli/ember-welcome-page/builds/697314709

File: tests/integration/components/welcome-page-test.js [hunk 1]
```diff
@@ -1,60 +1,58 @@
-import { VERSION } from '@ember/version';
-import { moduleForComponent, test } from 'ember-qunit';
+/* eslint-disable ember/new-module-imports */
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
```
Comment (ijlee2) [inline tests/integration/components/welcome-page-test.js:6]: Can we update the import statement to a named import?
```suggestion
import { hbs } from 'ember-cli-htmlbars';
```

Comment (rwwagner90) [reply tests/integration/components/welcome-page-test.js:6]: Seems fine to me 👍

**Comment by rwwagner90** [comment]
@MelSumner I was thinking it's best to have 3 separate commits. This one fixes the build and modernizes, yours will add the a11y stuff, and then I will follow up with a third PR doing the updates for the production stuff. If that's okay with you?

**Pull request merged**